frontend-react/src/components/ManagePublicKey/ManagePublicKeyChooseSender.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from "react";
+import { Form, Dropdown, Button, FormGroup } from "@trussworks/react-uswds";
+
+import { useOrganizationSenders } from "../../hooks/UseOrganizationSenders";
+import Spinner from "../Spinner";
+
+export interface ManagePublicKeyChooseSenderProps {
+    onSenderSelect: (sender: string) => void;
+}
+
+export default function ManagePublicKeyChooseSender({
+    onSenderSelect,
+}: ManagePublicKeyChooseSenderProps) {
+    const { isLoading, senders } = useOrganizationSenders();
+    const [selectedSender, setSelectedSender] = useState(
+        senders?.length === 1 ? senders[0].name : ""
```

Comment:
I realize that `useOrganizationSenders` is called above this component, but we shouldn't build an assumption on that.  If this component were rendered anywhere else, `senders` wouldn't be loaded at this point so I think we can just default to `undefined`/`null`, and it'll be chosen automatically in [the effect below](https://github.com/CDCgov/prime-reportstream/pull/8731/files#diff-8e46f388cb0b202962a67f71e21169bf7b34cd92b20920f6e6579c274dda97b8R19).